wurst/Creeps/UnitSpawnSystem.wurst

```diff
@@ -24,9 +29,8 @@ function onSpawnTime()
 	if level mod 7 == 0 and specialWaves.getSize() > 0
 		specialWaves.dequeue().spawn()
 	else
-		if normalWaves.getSize() > 0
-			normalWaves.dequeue().spawn()
-		if level mod 4 == 0 and normalWaves.getSize() > 0
+		normalWaves.dequeue().spawn()
```

Comment:
check if it has members before dequeuing to avoid errors at the end of the game.